src/nova-components/CreateFavorites/resources/js/components/fields/TextField.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="field-container">
+    <div class="label-spacing">
+      <Label :label="label" :isRequired="isRequired" />
+    </div>
+    <div class="input-spacing">
+      <TextInput
+        :type="type"
+        :error="error"
+        :validationError="validationError"
+        :sameError="sameError"
+        :placeholder="placeholder"
+        :changeInput="changeInput"
+        ref="input"
+      />
+      <p v-if="error && validationError" class="error">
+        The {{ label }} field is required and must be a Url.
+      </p>
+      <p v-else-if="error" class="error">The {{ label }} field is required.</p>
+      <p v-else-if="validationError" class="error">
+        The {{ label }} field must be a Url.
+      </p>
+      <p v-else-if="sameError" class="error">
+        This {{ label }} is already used.
+      </p>
```

Comment:
Create component for error `<p>` eg: `<error-labal ...>`, And replace it by `<p class="error">`